build/commands/lib/util.js

```diff
@@ -386,6 +386,10 @@ const util = {
       const androidComponentsStylesResDest = path.join(config.srcDir, 'components', 'browser_ui', 'styles', 'android', 'java', 'res')
       const androidSafeBrowsingResSource = path.join(config.braveCoreDir, 'browser', 'safe_browsing', 'android', 'java', 'res')
       const androidSafeBrowsingResDest = path.join(config.srcDir, 'chrome', 'browser', 'safe_browsing', 'android', 'java', 'res')
+      const androidDownloadInternalResSource = path.join(config.braveCoreDir, 'browser', 'download', 'internal', 'android', 'java', 'res')
```

Comment:
do these files have to be in the same directory or can we provide full paths in `brave_android_features_java_resources`, etc... in gn?

Reply:
cc: @samartnik

Reply:
But it is possible to do for the new resources, created a follow up issue for this https://github.com/brave/brave-browser/issues/32647